Hi, I need help about modudculab_ros

i did $catkin build modudculab_ros
and that i got this error 

/home/sol/catkin_ws/src/modudculab_ros/src/pub_setpoints_traj.cpp:74:40: error: ‘mavros_msgs::SetMode::Response {aka struct mavros_msgs::SetModeResponse_<std::allocator<void> >}’ has no member named ‘success’
                 offb_set_mode.response.success){

--- a/src/pub_setpoints_traj.cpp
+++ b/src/pub_setpoints_traj.cpp
@@ -71,7 +71,7 @@ int main(int argc, char **argv)
         if( current_state.mode != "OFFBOARD" &&
             (ros::Time::now() - last_request > ros::Duration(5.0))){
             if( set_mode_client.call(offb_set_mode) &&
-                offb_set_mode.response.success){
+                offb_set_mode.response.mode_sent){
                 ROS_INFO("Offboard enabled");
             }
             last_request = ros::Time::now();

